src/EditorFeatures/TestUtilities/QuickInfo/AbstractSemanticQuickInfoSourceTests.cs

```diff
@@ -49,52 +34,66 @@ internal Action<QuickInfoItem> WarningGlyph(Glyph expectedGlyph)
             return SymbolGlyph(expectedGlyph);
         }
 
-        internal void AssertSection(
+        private void AssertSection(
             string expectedText,
             ImmutableArray<QuickInfoSection> sections,
-            string textBlockKind,
-            FormattedClassification[] expectedClassifications = null)
+            string textBlockKind)
         {
             var textBlock = sections.FirstOrDefault(tb => tb.Kind == textBlockKind);
-            var text = textBlock != null ? textBlock.TaggedParts : ImmutableArray<TaggedText>.Empty;
-            AssertTaggedText(expectedText, text, expectedClassifications);
+            var taggedText = textBlock != null ? textBlock.TaggedParts : ImmutableArray<TaggedText>.Empty;
+
+            Assert.Equal(expectedText, taggedText.GetFullText());
         }
 
-        protected void AssertTaggedText(
-            string expectedText,
-            ImmutableArray<TaggedText> taggedText,
-            FormattedClassification[] expectedClassifications = null)
```

Comment:
The `expectedClassifications` parameter was unused and existing tests that used it were ineffectual.